### PR TITLE
GIX-1834: Remove login page

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Before changing the landing pages with new assets, we need to remove the Login page and its assets.

This is a requirement before merging #3169 

# Changes

* Remove Login page and related components.
* Remove layout for (login) route.
* Redirect to `/accounts` in (login) route page.

# Tests

* Remove layout.spec for (login) route.
* Add test for (login) route page.
* Remove Login.spec
* Fix e2e tests to start at `/accounts` directly.

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @dskloetd please review

I deployed it to https://kbruq-4yaaa-aaaaa-aacta-cai.nnsdapp.dfinity.network if you want to try it out.

**Comment by peterpeterparker** [comment]
Looks good 👍

The `goto` makes the login feels "slow" but we will tacklet that in #3169 so not a blocker for this PR.

goto

**Comment by lmuntaner** [comment]
> Looks good 👍
> 
> The `goto` makes the login feels "slow" but we will tacklet that in #3169 so not a blocker for this PR.
> 

I agree. On testnet this is even more accute.

The good thing is that after this PR we can work in parallel to improve the `/` experience and the new landing pages.

I'm also leaning more towards showing the accounts page there. But only for non-logged in users.

Could we move it inside `(app)` though @peterpeterparker ? To avoid duplicating the layout logic.

**Review by peterpeterparker — COMMENTED** [review]
@lmuntaner this screenshot is white https://github.com/dfinity/nns-dapp/blob/08ef3d011f72ca1a725c8f5c13dae1df32b80887/frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Login-1-Google-Chrome-darwin.png 

does `design.spec.ts` need more adjustements?

**Comment by peterpeterparker** [comment]
> Could we move it inside (app) though @peterpeterparker ? To avoid duplicating the layout logic.

Not sure because it might need a layout that should not be inherited but, we will find an acceptable solution. We can have a look together aftewards in the other PR.

**Comment by lmuntaner** [comment]
> @lmuntaner this screenshot is white https://github.com/dfinity/nns-dapp/blob/08ef3d011f72ca1a725c8f5c13dae1df32b80887/frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Login-1-Google-Chrome-darwin.png
> 
> does `design.spec.ts` need more adjustements?

It seems so, let me take a look. I guess that it should wait for the stuff to be loaded, but I wonder why it was working before.

**Comment by lmuntaner** [comment]
@dskloetd @peterpeterparker please review.

Fixing the screenshot test, but the rest is ready for review

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks a lot!

Ping me if you need me for the next PR

File: frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Signed-in-My-Tokens-1-Google-Chrome-linux.png
Comment (dskloetd) [inline frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Signed-in-My-Tokens-1-Google-Chrome-linux.png:1]: Why does it no longer say "My Tokens" at the top?

File: frontend/src/tests/e2e/images.spec.ts [hunk 1]
```diff
@@ -29,9 +29,9 @@ const expectImagesLoaded = async ({ page, sources }) => {
   }, sources.length);
 };
 
-test("Test images load on home page", async ({ page, context }) => {
-  await page.goto("/");
-  await expect(page).toHaveTitle("NNS Dapp");
+test.skip("Test images load on home page", async ({ page, context }) => {
```
Comment (dskloetd) [inline frontend/src/tests/e2e/images.spec.ts:32]: Why is this skipped? Should it have been removed instead or changed to test something else?

File: frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Login-1-Google-Chrome-linux.png
Comment (dskloetd) [inline frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Login-1-Google-Chrome-linux.png:1]: Why is this file deleted? Looking at the test it looks like it should now show the accounts tab with the login button, no?

File: frontend/src/tests/routes/login/page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+import { AppPath } from "$lib/constants/routes.constants";
+import { pageStore } from "$lib/derived/page.derived";
+import App from "$routes/(login)/+page.svelte";
+import { render, waitFor } from "@testing-library/svelte";
+import { get } from "svelte/store";
+
+describe("Layout", () => {
+  afterAll(() => {
```
Comment (dskloetd) [inline frontend/src/tests/routes/login/page.spec.ts:12]: Why `afterAll`?
You never need to do clean up in `afterAll` because no test runs after it.
`afterAll` is only to check if the tests didn't do anything they shouldn't have been doing.

File: frontend/src/tests/e2e/images.spec.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/e2e/images.spec.ts:32]: Oh, no! I was trying things and I forgot to remove this.

We should add a lint rule to not have skipped tests.

File: frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Login-1-Google-Chrome-linux.png
Comment (lmuntaner) [reply frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Login-1-Google-Chrome-linux.png:1]: Yes, correct. Weird. I'll look into the e2e tests.

File: frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Signed-in-My-Tokens-1-Google-Chrome-linux.png
Comment (lmuntaner) [reply frontend/src/tests/e2e/screenshots/design.spec.ts-Design-Signed-in-My-Tokens-1-Google-Chrome-linux.png:1]: That doesn't make sense either. I'll take a look.

File: frontend/src/tests/routes/login/page.spec.ts (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/tests/routes/login/page.spec.ts:12]: I copied from another test. I'll change it.
